binary/grakn

```diff
@@ -23,7 +23,7 @@ GRAKN_HOME=$(cd "$(dirname "${path}")" && pwd -P)
 GRAKN_CONFIG="server/conf/grakn.properties"
 
 CONSOLE_JAR_FILENAME=(${GRAKN_HOME}/console/services/lib/io-grakn-console-grakn-console*.jar)
-SERVER_JAR_FILENAME=(${GRAKN_HOME}/server/services/lib/io-grakn-core-grakn-server*.jar)
+SERVER_JAR_FILENAME=(${GRAKN_HOME}/server/lib/grakn-core-grakn-core-server*.jar)
```

Comment:
This change should never have happened, @lolski . You only needed to make this change because of the mistake we made here: https://github.com/graknlabs/grakn-2.0/pull/7#pullrequestreview-478152772

But this should be a code-smell to pay attention to next time, @lolski; if we have to rename something that used to work all this while, perhaps something just broke to cause this change? That would have helped us notice that we named the artifacts incorrectly in Grakn Core 2.0.